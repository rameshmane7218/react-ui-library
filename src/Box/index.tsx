export { Box, BoxProps } from './Box';
