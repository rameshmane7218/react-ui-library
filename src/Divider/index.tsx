export { Divider, HrProps } from './Divider';
