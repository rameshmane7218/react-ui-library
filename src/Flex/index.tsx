export { Flex, FlexProps } from './Flex';
