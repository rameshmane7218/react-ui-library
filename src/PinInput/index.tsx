export { InputField, InputFieldProps } from './InputField';
export { PinInput, PinInputProps } from './PinInput';
export { PinInputExample } from './PinInputExample';
