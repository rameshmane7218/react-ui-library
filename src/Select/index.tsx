export { Select, SelectProps } from './Select';
export { Option, OptionProps } from './Option';
