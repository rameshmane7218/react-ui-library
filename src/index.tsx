export * from './Button';
export * from './Thing';
