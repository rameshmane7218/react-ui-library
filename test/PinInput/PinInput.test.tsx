import React from 'react';
import { render } from '@testing-library/react';
import { PinInput } from '../../src';
describe('PinInput Component', () => {
  it('should render PinInput component without crashing', () => {
    const { asFragment } = render(<PinInput limitPerBox={1} length={4} />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should have correct default value', () => {
    const { asFragment } = render(
      <PinInput limitPerBox={2} length={4} defaultValue={'12345678'} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
