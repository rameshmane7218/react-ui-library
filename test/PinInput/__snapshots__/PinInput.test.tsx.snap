// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinInput Component should have correct default value 1`] = `
<DocumentFragment>
  <div
    class="sc-iBYQkv euSEBe"
  >
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="2"
      placeholder="〇"
      type="text"
      value="12"
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="2"
      placeholder="〇"
      type="text"
      value="34"
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="2"
      placeholder="〇"
      type="text"
      value="56"
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="2"
      placeholder="〇"
      type="text"
      value="78"
    />
  </div>
</DocumentFragment>
`;

exports[`PinInput Component should render PinInput component without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-iBYQkv euSEBe"
  >
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="1"
      placeholder="〇"
      type="text"
      value=""
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="1"
      placeholder="〇"
      type="text"
      value=""
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="1"
      placeholder="〇"
      type="text"
      value=""
    />
    <input
      class="sc-gKPRtg EKkfY"
      maxlength="1"
      placeholder="〇"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
